api/game/clueless.py

```diff
@@ -192,14 +193,28 @@ def initialize_board(self) -> Dict:
         # front-end Cards.jsx uses cards_to_display only -- not separately displaying visible + player cards
         self.state["cards_to_display"] = {}
         for k, v in self.state["player_cards"].items():
-            self.state["cards_to_display"][k] = self.state["player_cards"][k] + self.state["visible_cards"]
+            self.state["cards_to_display"][k] = self.state["player_cards"][
+                k] + self.state["visible_cards"]
+
+        # Build lists of cards for suggestion/accusation
+        self.state["selectable_cards"]["rooms"] = self.rooms

```

Comment:
```suggestion
        self.state["selectable_cards"]["rooms"] = list(set(self.rooms).difference(set(self.state["visible_cards"])))
```
and similar for suspects, weapons if you want to cut a few lines